README.md

```diff
@@ -1,3 +1,7 @@
+[![Dart CI](https://github.com/dart-lang/pool/actions/workflows/test-package.yml/badge.svg)](https://github.com/dart-lang/pool/actions/workflows/test-package.yml)
```

Comment:
Need to fix this link! https://github.com/dart-lang/pool/actions/workflows/ci.yml

The other one is dead!

Reply:
good catch - will do